src/main/java/com/hubspot/jinjava/el/ext/JinjavaBeanELResolver.java

```diff
@@ -100,7 +104,15 @@ public Object invoke(
     Class<?>[] paramTypes,
     Object[] params
   ) {
-    if (method == null || RESTRICTED_METHODS.contains(method.toString())) {
+    JinjavaInterpreter interpreter = (JinjavaInterpreter) context
+      .getELResolver()
+      .getValue(context, null, ExtendedParser.INTERPRETER);
```

Comment:
Can you use `JinjavaInterpreter.getCurrent()` instead of using the resolver to get the interpreter? I think it might be faster.

Reply:
I changed to `JinjavaInterpreter.getCurrent()`,